src/firebase/config.js

```diff
@@ -0,0 +1,18 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+  apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
+  authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
+};
+
+const app = initializeApp(firebaseConfig);
+const firebaseAuth = getAuth(app);
+const firebaseDb = getFirestore(app);
```

Comment:
Wouldn't you make use of the storage bucket as well? To store media assets? Like profile photos, and community banners?

Reply:
Also, you created an unrelated button although the PR is "Firebase init". Maybe we should try to avoid that in the future

Reply:
@annonymousauthority the button is from the merge pull request from main branch